pom.xml

```diff
@@ -162,7 +162,7 @@
     <commons.math3.version>3.4.1</commons.math3.version>
     <!-- managed up from 3.2.1 for SPARK-11652 -->
     <commons.collections.version>3.2.2</commons.collections.version>
-    <scala.version>2.12.10</scala.version>
+    <scala.version>2.12.14</scala.version>
```

Comment:
@dongjoon-hyun please modify the scala version to 2.12.14 in profile `scala-2.12`, otherwise there maybe a `Error:scala: bad option: -P:silencer:globalFilters=.*deprecated.*` error when tick the profile `scala-2.12`.

Reply:
Oh, thanks. I missed there.